src/main/java/picard/vcf/MendelianViolations/FindMendelianViolations.java

```diff
@@ -173,8 +173,9 @@
     @Argument(doc = "List of possible names for female sex chromosome(s)")
     public Set<String> FEMALE_CHROMS = CollectionUtil.makeSet("X", "chrX");
 
-    @Argument(doc = "List of chr:start-end for pseudo-autosomal regions on the female sex chromosome. Defaults to HG19/b37 & HG38 coordinates.")
-    public Set<String> PSEUDO_AUTOSOMAL_REGIONS = CollectionUtil.makeSet("X:10001-2649520", "X:59034050-59373566", "chrX:10000-2781479", "chrX:155701382-156030895");
+    @Argument(doc = "List of chr:start-end for pseudo-autosomal regions on the female sex chromosome. Defaults to HG19/b37 & HG38 coordinates.",
+    shortName = "PAR")
+    public Set<String> PSEUDO_AUTOSOMAL_REGIONS = CollectionUtil.makeSet("X:60001-2699520", "X:154931044-155260560", "chrX:10000-2781479", "chrX:155701383-156030895");
```

Comment:
```suggestion
    public Set<String> PSEUDO_AUTOSOMAL_REGIONS = CollectionUtil.makeSet("X:60001-2699520", "X:154931044-155260560", "chrX:10001-2781479", "chrX:155701383-156030895");
```
may as well fix hg38 as well